app/server/utils/user.py

```diff
@@ -687,6 +687,16 @@ def proccess_create_or_modify_user_request(
     # Location fires an async task that needs to know user ID
     db.session.flush()
 
+    if attribute_dict.get('secondary_user_first_name'):
+        create_transfer_account_user(
```

Comment:
Should the secondary user have an accompanying `set_custom_attributes` like the primary user does?

Reply:
I decided against it, because the custom attributes can really only be inferred about the primary user (for example we have no idea if user two is female or not).